django/contrib/admin/options.py

```diff
@@ -1744,6 +1745,9 @@ def __init__(self, parent_model, admin_site):
     def media(self):
         extra = '' if settings.DEBUG else '.min'
         js = ['jquery%s.js' % extra, 'jquery.init.js', 'inlines%s.js' % extra]
+        # TODO: Was removed in master, modified in patch.
```

Comment:
I do not see where this is added above. The previous line does not add it, so I'm wondering if there is something else missing.